### DISCUSSION TITLE
Translation Issue

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#6434] [Q&A]
I am adding new feature to switch between en and ar lang files, i just added ar.json and ar folder as i read the doc and add link in the navbar include [en , ar]. when i sitch to en it is okay but when i switch to ar and external links i have added in the side menu go well but any side menu related to statamic like : users , groups or permissions and so on not work, how to make statamic sidemenu nav read from arabic also ? Please advice

**Comment by jasonvarga** [comment]
> I am adding new feature to switch between en and ar lang files

What does this mean?
